src/lib/structures/game/BinbGame.ts

```diff
@@ -22,6 +22,7 @@ import { setTimeout } from 'node:timers/promises';
 
 export class BinbGame extends Game {
 	public readonly gameType = GameType.Binb;
+	public override readonly acceptedAnswer: AcceptedAnswer = AcceptedAnswer.Both;
```

Comment:
Does the explicit type need to be there? Also, you should make a warning in `start.ts` if accepted answer is set to something other than Both and it's the binb gamemode (maybe add it in the description it's always both? idk?)
```suggestion
	public override readonly acceptedAnswer = AcceptedAnswer.Both;
```